### ISSUE TITLE
Convert CI to GitHub Actions

### ISSUE THREAD

**Issue by jacebrowning** [jacebrowning/template-python#214]

**Comment by kyluca** [comment]
Hey Jace, I see you've self-assigned this issue but I'm sure you're very busy. Wondering if you would be happy to take a Hacktoberfest PR for this?

I can see the cookiecutter project gets a GHA workflow here: https://github.com/jacebrowning/template-python/blob/main/%7B%7Bcookiecutter.project_name%7D%7D/.github/workflows/main.yml

So I'm guessing this issue is talking about the template project itself, which uses Travis: https://github.com/jacebrowning/template-python/blob/main/.travis.yml

**Comment by jacebrowning** [comment]
@kyluca yes, I'd like to port the template over to use GitHub Actions. Thanks for the offer to help!
